webpack.config.base.js

```diff
@@ -19,8 +19,8 @@ module.exports = (options) => ({
   context: path.resolve(__dirname, 'src'),
   resolve: {
     modules: [
-      path.resolve('./src'),
-      path.resolve('./node_modules')
+      path.resolve(__dirname, 'src'),
+      'node_modules'
```

Comment:
I must have changed this because of an issue with using absolute paths for the import statements. However, if this works fine , then sounds good.